grade-scripts/trackers-from-requests.js

```diff
@@ -41,6 +42,8 @@ const run = async () => {
     let siteDataArray = scriptUtils.getSiteData(inputPath, fileForSubset)
 
     for (let siteData of siteDataArray) {
+        if (siteData.failed) continue
```

Comment:
Sort of unrelated, just noticed it breaking with some of the request data I had lying around.